downloadBookImages.sh

```diff
@@ -5,8 +5,20 @@ height=$(($width*2))
 
 rm -f src/main/resources/com/vaadin/exampledata/BookImageBackground.txt
 for baseurl in `cat src/main/resources/com/vaadin/exampledata/BookImageBackground.src`;
-do 
+do
   url="$baseurl?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=$width&h=$height&q=80"
   echo "Downloading $url"
   (echo -n "data:image/png;base64," ; curl -s "$url"|base64  ) >>  src/main/resources/com/vaadin/exampledata/BookImageBackground.txt
 done
+
+rm -f src/main/resources/com/vaadin/exampledata/BookImageFiles/*
+rm -f src/main/resources/com/vaadin/exampledata/BookImageFileNames.txt
+for baseurl in `cat src/main/resources/com/vaadin/exampledata/BookImageBackground.src`;
+do
+  [[ "$baseurl" =~ [^\/]+$ ]]
+  filename="${BASH_REMATCH[0]}.jpg"
+  url="$baseurl?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=$width&h=$height&q=80&fm=jpg"
```

Comment:
Why do we need to download the same files as jpg? Can't we use the pngs?

Reply:
Because, for example, https://images.unsplash.com/photo-1515150144380-bca9f1650ed9 is:
jpg: 29KB
png: 165KB

Reply:
So can we use only jpgs then?

Reply:
I removed `BookImageBackground.txt`, and all the related logic. Now Data URLs are being generated from the jpgs.